src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -336,7 +338,12 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
         </h4>
         <input type="hidden" name="name" ref={register} value={values.name} />
         <span className="input-info">
-          The URL of your venue will be: <b>{urlSafeName}</b>
+          The URL of your venue will be:{" "}
+          <b>
+            {venueId
+              ? `${window.location.host}${venueLandingUrl(venueId)}`
+              : urlSafeName}
+          </b>
```

Comment:
This doesn't seem like a script change..?

Reply:
Oh looks like it's not. Not sure how this got in.

Since you're planing to recreate the PR or direct this work toward a different change, you can exclude this stuff and it'll be all good.